hack/installers/install-lint-tools.sh

```diff
@@ -2,6 +2,6 @@
 set -eux -o pipefail
 
 # renovate: datasource=go packageName=github.com/golangci/golangci-lint
-GOLANGCI_LINT_VERSION=1.61.0
+GOLANGCI_LINT_VERSION=v1.61.0
 
 GO111MODULE=on go install "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}"
```

Comment:
It was my fault :
- https://github.com/argoproj/argo-cd/commit/d17aafd19a465fb9992d8189381adc3bdd488ead#diff-379bb3898b60010b7127603fcc9dcfb4d7f8e921997c1b43eaba78c5a99ce3c8L4  

The `v` should be outside the VM else `renovate` will either fail or won't stop removing it.
```diff
- GOLANGCI_LINT_VERSION=v1.61.0
+ GOLANGCI_LINT_VERSION=1.61.0

- GO111MODULE=on go install "github.com/golangci/golangci-lint/cmd/golangci-lint@${GOLANGCI_LINT_VERSION}"
+ GO111MODULE=on go install "github.com/golangci/golangci-lint/cmd/golangci-lint@v${GOLANGCI_LINT_VERSION}"

Reply:
You're right, "v" should be outside.
Thanks

Reply:
thank you for fixing it